src/frontend/src/icp/utils/icrc.utils.ts

```diff
@@ -133,4 +133,4 @@ export const buildIcrcCustomTokenMetadataPseudoResponse = ({
 };
 
 export const isIcrcCustomToken = (token: Partial<IcrcCustomToken>): token is IcrcCustomToken =>
-	token.standard === 'icrc' && 'enabled' in token;
+	(token.standard === 'icp' || token.standard === 'icrc') && 'enabled' in token;
```

Comment:
- I would rename the function to `isCustomTokenIC` then
- You can maybe shorten `["icp", "icrc"].includes(token.standard)` (if interesting and not sure TS likes it)?

Reply:
yes on the first, on the second shall be `["icp", "icrc"].includes(token.standard ?? '')` since it am getting the partial `IcrcCustomToken`

still ok?

Reply:
I wonder if that will be solved with TypeScript 5.5 but, let's keep your implementation. Thanks for checking.